Procfile

```diff
@@ -1,4 +1,4 @@
-web: ./script/server-pgbouncer
+web: ./script/server-pgbouncer && ./script/appsignal
```

Comment:
I don't think this line does anything, since `script/server-pgbouncer` blocks the process forever and it will never reach `script/appsignal`. as per the other comment, I am not sure we need `script/appsignal`.